api-remove.go

```diff
@@ -213,6 +213,10 @@ type RemoveObjectError struct {
 	Err        error
 }
 
+func (err *RemoveObjectError) Error() string {
```

Comment:
```suggestion
func (err *RemoveObjectError) Error() string {
     if err.Err != nil {
         return err.Err.Error()
     }
     return "<nil>"
```

Reply:
`err.Err` currently would never be nil and should not be nil in the future if we implement it as an error, but agree with we should check about it.